### ISSUE TITLE
Problems with refreshWeekView() after modify numOfDays

### ISSUE THREAD

**Issue by aron1127** [zjfjack/JZCalendarWeekView#77]
I have created a button that can control numofDays.

I pressed the button and ran that code.

```
reloadnumDay += 1
calendarWeekView.numOfDays = reloadNumDay
calendarWeekView.refreshWeekView()
```
But I did not get the result I wanted.

Initial Screen:
![first](https://user-images.githubusercontent.com/28154360/58081774-98f3e900-7bf0-11e9-8a09-418bc5463f45.png)

Result screen:
![sec](https://user-images.githubusercontent.com/28154360/58081800-a4471480-7bf0-11e9-950c-29324db34cea.png)

The screen I want:
![thi](https://user-images.githubusercontent.com/28154360/58081817-ac06b900-7bf0-11e9-8442-e4af86d21bdd.png)

It should start at 1 but start at 2 in the result screen.

I do not know why these results come out. Is there a workaround?

**Comment by zjfjack** [comment]
Hi, currently I didn't handle this requirement. Do you want to always keep the same day when you change the `numOfDays` ?

**Comment by aron1127** [comment]
Yes. When adding or subtracting numofDays, I want to keep the same day

**Comment by zjfjack** [comment]
What if it is 7 days with `firstDayOfWeek`?

**Comment by aron1127** [comment]
I always set it to Monday.

**Comment by aron1127** [comment]
Hi. I've fixed the problem by modifying your library.

I created a `numofDaysReload` variable in JZBaseWeekView.swift and used it in the `updateWeekView()` function.

JZBaseWeekView.swift : 
```
public var numOfDays: Int! 
public var numofDaysReload: Int = 1
...

open func refreshWeekView() 
{
    updateWeekView(to: self.initDate.add(component: .day, value: numOfDays + numofDaysReload))
}
```

In ViewController :

```
if numofDays += 1
{
    calendarWeekView.numofDaysReload = -1
}
else if numofDays -= 1
{
    calendarWeekView.numofDaysReload = 1
}
calendarWeekView.refreshWeekView()
```
I do not think it's an efficient method, but it's solved.
I am a junior developer in Korea and have poor English and coding skills. Sorry for bothering you.
If you see my reply, please close the issue.

**Comment by zjfjack** [comment]
Hi,

Thanks for doing it first. I am currently very busy. If I have time in the future and I might give you a better way to achieve this.

Thanks
